dataprofiler/profilers/utils.py

```diff
@@ -344,15 +344,15 @@ def biased_kurt(df_series: Series) -> float:
     :param df_series: data to get kurtosis of, assuming floats
     :type df_series: pandas Series
     :return: biased kurtosis
-    :rtype: float
+    :rtype: np.float64
     """
     n = len(df_series)
     if n < 1:
-        return np.nan
+        return np.float64(np.nan)
```

Comment:
required for `assert_profile_equal` in testing ... otherwise could be a float64 & float